src/pages/components/card-group.mdx

```diff
@@ -40,44 +40,77 @@ Card group by default arranges cards on a collapsed grid, and applies a 1px bord
 
 ### Card group with card in card
 
-Card in card can be used with the card group component to give focus to the first card.
+Card in card can be used with the card group component to give focus to the first card. This works well when you need to give prominence to a single card in the group, with other cards providing secondary resources.
 
 <Row>
+
 <Column colMd={8} colLg={8}>
 
-![Image of Card in card component](../../images/component/card-group/card-group-2.png)
+![Image of Card in card component above card group](../../images/component/card-group/card-group-variations-01.png)
 
 <Caption>
-  Card group with card in card above. The example shown here is using the narrow
-  grid.
+  Card group with card in card above, all cards using the condensed grid
 </Caption>
 
 </Column>
+
 </Row>
 
 ## Modifiers
 
-Card group allows the following properties to be modified to achieve a wide range of designs.
-
 ### Cards per row
 
-Cards per row dictates the number of cards in a row. Sometimes this can be adjusted based on total card number or breakpoint to avoid orphaning single card in the last row.
+Cards per row dictates the number of cards in a given row in the group. Depending on the total number of cards, this can be adjusted to avoid orphaning a single card in the last row.
```

Comment:
```suggestion
Cards per row dictates the number of cards in a given row in the group. Depending on the total number of cards, this can be adjusted to avoid orphaning a single card in the last row. 

_Note: card groups should always adhere to the 2x grid and therefore we do not recommend selecting an odd number of cards per row (ie 5). In general 2–4 cards works best with most Carbon for IBM.com layout configurations._ 
```